examples/with-realm-web/pages/index.js

```diff
@@ -0,0 +1,80 @@
+import { FIND_MOVIES } from '../lib/graphql-operations'
+import { APP_ID } from '../lib/RealmClient'
+import { useQuery } from '@apollo/react-hooks'
+
+const IndexPage = () => {
+  const { loading, error, data } = useQuery(FIND_MOVIES, {
+    query: { year: 2014, rated: 'PG' },
+  })
+
+  if (error) {
+    console.log(error.message)
+  }
+
+  const movies = data ? data.movies : null
+
+  return (
+    <>
+      <div className="App">
+        <h1>"PG" Rated Movies - 2014</h1>
+
+        {APP_ID === 'realm-example-bspbt' ? (
+          <div className="status">Replace REALM_APP_ID with your App ID</div>
+        ) : (
+          !loading && !movies && <div className="status">No data Found!</div>
+        )}
```

Comment:
This should be part of the readme, if the instructions are clear we don't need to do any additional checks here.

Reply:
Yup will highlight this part in the readme.

Reply:
@darshkpatel Remove this check, leave only the second part:
```
!loading && !movies && <div className="status">No data Found!</div>
```